client/landing/stepper/declarative-flow/internals/steps.tsx

```diff
@@ -321,6 +321,28 @@ export const STEPS = {
 		slug: 'platform-identification',
 		asyncComponent: () => import( './steps-repository/platform-identification' ),
 	},
+	UNIFIED_DOMAINS: {
```

Comment:
This change also feels like it could be split out fairly easily.

Reply:
Yes, but it was needed to debug this PR without losing my mind 😄  I needed the chunk names to tell what was loaded when. 

Moved to https://github.com/Automattic/wp-calypso/pull/100063